gammapy/data/tests/test_observation_summary.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
+from astropy.coordinates import SkyCoord
+
+from ...data import DataStore, ObservationTable, ObservationTableSummary
+
+from ...utils.testing import data_manager, requires_data, requires_dependency
+
+@requires_data('gammapy-extra')
+@pytest.fixture
+def summary():    
+    data_store = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
+    target_pos = SkyCoord(83.633083, 22.0145, unit='deg')
+    return ObservationTableSummary(data_store.obs_table, target_pos)
+
+@requires_data('gammapy-extra')
+def test_str():
+    text = str(summary())
+    assert 'Observation summary' in text
+
+@requires_data('gammapy-extra')
```

Comment:
I think if you declare `@requires_data('gammapy-extra')` on the fixture, you don't need to declare it on the test functions that use the fixture. So please remove this on the test functions.